simulation-api/src/main/java/com/powsybl/simulation/ImpactAnalysisTool.java

```diff
@@ -128,21 +126,25 @@ private static String okToStr(Boolean b) {
     }
 
     private static void prettyPrint(Multimap<String, SecurityIndex> securityIndexesPerContingency, PrintStream out) {
-        Table table = new Table(1 + SecurityIndexType.values().length, BorderStyle.CLASSIC_WIDE);
-        table.addCell("Contingency");
-        for (SecurityIndexType securityIndexType : SecurityIndexType.values()) {
-            table.addCell(securityIndexType.toString());
-        }
+        try (Writer myWriter = new StringWriter();
+             AbstractTableFormatter formatter = new AsciiTableFormatter(myWriter, "myFormatter", 1 + SecurityIndexType.values().length)) {
+            formatter.writeCell("Contingency");
```

Comment:
It seems to be the column header. Use the constructor to define the columns

Reply:
same response as before.